### PR TITLE
Add keyword search redesign to homepage

### PR DESCRIPTION
**Pull request by mjoyce91**
Adds the redesigned search bar to the homepage, which accepts keyword searches and upon submission, directs the user to the results page with that keyword. Makes use of the same `ResultsSearchHeader` component as the Results page for consistency (though some styles differ between the home and results pages). Clears out old filters to prep for additional redesign tasks.

### CODE REVIEW

File: src/Containers/Home/Home.jsx [hunk 1]
```diff
@@ -50,7 +49,7 @@ class Home extends Component {
 Home.propTypes = {
   onNavigateTo: PropTypes.func.isRequired,
   fetchData: PropTypes.func,
-  isLoading: PropTypes.bool,
+  // isLoading: PropTypes.bool,
```
Comment (burgwyn) [inline src/Containers/Home/Home.jsx]: Remove

File: src/sass/_resultsSearchBarContainer.scss [hunk 2]
```diff
@@ -45,25 +46,22 @@
     label {
       display: block;
       text-align: left;
+      text-transform: uppercase;
     }
   }
 
-  .search-location {
-    padding: 0 30px;
+  .usa-button {
```
Comment (burgwyn) [inline src/sass/_resultsSearchBarContainer.scss]: Are we overriding a USWDS style here?  Should we use a `tm-` instance instead?

File: src/sass/_variables.scss [hunk 3]
```diff
@@ -7,6 +7,7 @@ $blue-primary: #0071bc;
 $color-white: #fff;
 $color-black: #000;
 $color-gray: #5b616b;
+$tm-dark-blue: #00293f;
```
Comment (burgwyn) [inline src/sass/_variables.scss]: Is the thinking that we'll identify our custom colors as `tm-`?

File: src/Components/Homepage/Homepage.jsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
```
Comment (burgwyn) [inline src/Components/Homepage/Homepage.jsx]: Let's move this from Components to Containers

File: src/sass/_resultsSearchBarContainer.scss (hunk 2 above)
Comment (mjoyce91) [reply src/sass/_resultsSearchBarContainer.scss]: This is only within the context of the `results-search-bar-container` class.

File: src/sass/_variables.scss (hunk 3 above)
Comment (mjoyce91) [reply src/sass/_variables.scss]: Right, opposed to the generic USWDS colors.

**Pull request merged**